install/docker-compose.base.yml

```diff
@@ -24,3 +24,4 @@ services:
       - FOSSOLOGY_DB_HOST=fossology-db
       - FOSSOLOGY_DB_NAME=fossology
       - FOSSOLOGY_DB_USER=fossy
+    user: fossy
```

Comment:
it's not working yet. The only way to test travis-ci is to commit. No review necessary yet.
I will always cleanup, before this needs to be reviewed again.